lib/diagnostic.js

```diff
@@ -42,6 +80,14 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.deleteOne(
```

Comment:
This will try to delete a document that has the matching key and value pairs passed in.  In this case, it won't find anything.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```